app/src/main/java/org/simple/clinic/signature/SignatureActivity.kt

```diff
@@ -0,0 +1,12 @@
+package org.simple.clinic.signature
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import org.simple.clinic.R
+
+class SignatureActivity : AppCompatActivity() {
```

Comment:
Instead of `Activity`, why aren't we just using a Dialog? 🤔

Reply:
@vinaysshenoy can you help me here? I asked the same question and remember vaguely, not sure about it.

Reply:
The reason is that I am not sure if a dialog will support landscape mode if the parent activity it is running in does not.

Reply:
Now that we're keeping the screen orientation as portrait, this could have worked, right?

Reply:
We should support landscape mode for signature signing since smaller width screens might not give enough real estate for a complete signature.

Reply:
Just a thought for later, instead of just moving the signature screen to a separate activity. Should we consider moving the entire doctor deep link flow to a different activity? 🤔 

One downside for this might be breaking the single activity approach we have right now.